### DISCUSSION TITLE
LDM Notes for June 24th, 2024

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#8262] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2024/LDM-2024-06-24.md

## Agenda

- First-Class Spans
- `field` questions
    - Mixing auto-accessors
    - Nullability

**Comment by dersia** [comment]
for first class spans, considering the decision that was made in the ldm meeting, wouldn't it make sense to also include `TensorSpan` and `ReadOnlyTensorSpan` together with `System.Span` and `System.ReadOnlySpan`?
